algebird-core/src/main/scala/com/twitter/algebird/HyperLogLog.scala

```diff
@@ -79,34 +80,65 @@ object HyperLogLog {
   @inline
   def twopow(i: Int): Double = java.lang.Math.pow(2.0, i)
 
+  @deprecated("this is no longer used", since = "1.12.3")
```

Comment:
May be nice to provide an alternative here: `use j(Array[Byte], Int) instead` or something.